js/utils/draggable_background.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * Draggable Background plugin for jQuery
+ *
+ * v1.2.4
+ *
+ * Copyright (c) 2014 Kenneth Chung
+ *
+ * Licensed under the MIT license:
+ *   http://www.opensource.org/licenses/mit-license.php
+ */
+
+/* eslint-disable */
```

Comment:
It's old, and not available from npm. It's de-linted because it's a placeholder, it's going to need some tweaking or to just be replaced. 

I can move it to the lib folder until then.